src/FileUrlGeneratorDecorator.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\localgov_moderngov_tpl;
+
+use Drupal\Core\File\FileUrlGeneratorInterface;
+use Drupal\Core\Url;
+
+/**
+ * Decorates the file_url_generator service.
+ *
+ * On the ModernGov template page, requests for relative URL generation is
+ * serviced by the absolute URL generation method.
+ */
+class FileUrlGeneratorDecorator implements FileUrlGeneratorInterface {
+
+  /**
+   * {@inheritdoc}
+   *
+   * When on the ModernGov tpl page, generates absolute URLs even for relative
```

Comment:
I LOVE this. Thanks!

Reply:
Yeah, this file_url_generator service is a [new feature](https://www.drupal.org/node/2940031) of Drupal 9.3.x that has made things so much easier for this ModernGov thing :)